src/sa_web/static/js/views/dataset-form-view.js

```diff
@@ -0,0 +1,214 @@
+/*globals _ Spinner Handlebars Backbone jQuery Gatekeeper */
+
+var Shareabouts = Shareabouts || {};
+
+(function(S, $, console){
+  S.DatasetFormView = Backbone.View.extend({
```

Comment:
To keep things DRY, we should probably have this extend `place-form-view.js`. We can do this by using `S.PlaceFormView.extend` instead of `Backbone.View.extend`